File: apps/android/app/src/main/java/com/roundsloyalty/app/data/models/Business.kt
package com.roundsloyalty.app.data.models

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class Business(
    val id: String,
    @SerialName("owner_id") val ownerId: String,
    val name: String,
    val description: String? = null,
    @SerialName("logo_url") val logoUrl: String? = null,
    val address: String? = null,
    val lat: Double? = null,
    val lng: Double? = null,
    @SerialName("qr_code_secret") val qrCodeSecret: String? = null,
)

File: apps/ios/RoundsLoyalty/Models/Business.swift
import Foundation

struct Business: Codable, Identifiable {
    let id: UUID
    let ownerId: UUID?
    var name: String
    var description: String?
    var logoUrl: String?
    var address: String?
    var lat: Double?
    var lng: Double?
    var qrCodeSecret: String?

    enum CodingKeys: String, CodingKey {
        case id
        case ownerId = "owner_id"
        case name
        case description
        case logoUrl = "logo_url"
        case address
        case lat
        case lng
        case qrCodeSecret = "qr_code_secret"
    }
}
